### DISCUSSION TITLE
What to do when there is a table in the answer section of the card

### DISCUSSION THREAD

**Discussion by jacks935** [st3v3nmw/obsidian-spaced-repetition#1005] [Q&A]
Is there any way to display a card with a table in the answer? In obsidian, there is a blank row in front of the table, which conflicts with the syntax of the plugin

**Comment by 4Source** [comment]
The blank row is new "feature" of obsidian if you switch back to about Obsidian 1.5 I think there wasn't this blank line. Other way Is to "destroy" the table with source mode in your note by removing the blank line. This will not result in a perfekt table but if you edit it after filling the table the spacing should be about right. 

This should be fixed in the future but at the moment these are the only options I could think of.

**Comment by ronzulu** [comment]
Being able to include blank lines within a flashcard is being addressed by PR:
[Add new option allowing for multiline cards with empty lines #1012 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/1012)
